proposal.md

```diff
@@ -101,13 +101,18 @@ are interested in incorporating into our web application.
 ## Timetable
 
 ## Facilities
+We will be running server-side code on a cloud hosting service. Hosts such as Amazon Web Services offer free trials for the first year, which would cover the length of this project. If these services are unavailable for free, we would expect our hosting costs to be less than $100.
```

Comment:
Like this is just a preference thing, but it'd be better if you put line breaks in where the paragraph wraps around so that the paragraphs aren't just super long lines.